Description: Ignore emoji pack in super-large emoji rendering
 It is necessary to build against the unpatched rlottie lib.
Bug-Samsung: https://github.com/Samsung/rlottie/pull/252
Author: Nicholas Guriev <guriev-ns@ya.ru>
Modified: Sat, 10 Aug 2019 11:30:57 +0300

--- a/Telegram/lib_lottie/lottie/lottie_animation.cpp
+++ b/Telegram/lib_lottie/lottie/lottie_animation.cpp
@@ -136,15 +136,7 @@ std::unique_ptr<rlottie::Animation> CreateFromContent(
 	const auto string = UnpackGzip(content);
 	Assert(string.size() <= kMaxFileSize);
 
-	auto result = rlottie::Animation::loadFromData(
-		string,
-		std::string(),
-		std::string(),
-		false,
-		(replacements
-			? replacements->replacements
-			: std::vector<std::pair<std::uint32_t, std::uint32_t>>()));
-	return result;
+	return rlottie::Animation::loadFromData(string, "");
 }
 
 } // namespace details
